Explorer updates (#4583)
fishtown-analytics/docs.getdbt.com: website/docs/docs/collaborate/model-performance.md

---
title: "Model performance"
sidebar_label: "Model performance"
description: "Learn about ."
---

> dbt Explorer provides metadata on dbt Cloud runs for in-depth model performance and quality analysis. This feature assists in reducing infrastructure costs and saving time for data teams by highlighting where to fine-tune projects and deployments &mdash; such as model refactoring or job configuration adjustments.

Reviewer: You might consider starting with the value (2nd sentence).

<LoomVideo id='98f33b3b7a374df0b7c04747eae6ef44' />

:::tip Public preview 

> The model performance feature is now available in dbt Explorer! Check it out! 

Reviewer: The second sentence might not be easily understood when English is someone's second language. Definitely bring on the excitement. Maybe "Go explore!" (ha ha)

:::

## The Performance overview page 

> The Performance overview page presents a comprehensive analysis across all project models, helping to pinpoint areas for performance enhancement. It displays the longest-running models, those most frequently executed, and the ones with the highest failure rates during runs/tests. Data can be segmented by environment and job type which can offer insights into:

Reviewer: I also wonder if this paragraph would benefit from bringing starting with the benefits. Maybe something like:
```suggestion
You can pinpoint areas for performance enhancement by using the Performance overview page. This page presents a comprehensive analysis across all project models and displays the longest-running models, those most frequently executed, and the ones with the highest failure rates during runs/tests. Data can be segmented by environment and job type which can offer insights into:
```

- Most executed models (total count).
- Models with the longest execution time (average duration).
- Models with the most failures, detailing run failures (percentage and count) and test failures (percentage and count).

Each data point links to individual models in Explorer. 

<Lightbox src="/img/docs/collaborate/dbt-explorer/example-performance-overview-page.png" width="80%" title="Example of Performance overview page"/>

You can view historical metadata for up to the past three months. Select the time horizon using the filter, which defaults to a two-week lookback.

<Lightbox src="/img/docs/collaborate/dbt-explorer/ex-2-week-default.png" title="Example of dropdown"/>

## The Model performance tab

> The Model performance tab allows for historical performance analysis, showing trends in execution times, counts, and failures. Daily execution data includes:

Reviewer: Same feedback here about bringing the value up front.

Author: sounds good!
